src/repl.jl

```diff
@@ -283,10 +283,12 @@ end
 NEW_KEYBINDINGS = create_keybindings()
 
 function insert_keybindings(repl = Base.active_repl)
-    mirepl = isdefined(repl,:mi) ? repl.mi : repl
+    mirepl = isdefined(repl,:mistate) ? repl.mistate : repl
     main_mode = mirepl.interface.modes[1]
-    p = mirepl.interface.modes[5]
+    php_idx = VERSION >= v"1.11.0-rc1" ? 6 : 5
+    p = mirepl.interface.modes[php_idx]::LineEdit.PrefixHistoryPrompt
```

Comment:
This seems brittle. Better to find first?

Reply:
Thanks. I have improved the implementation by using `findfirst`. (https://github.com/KristofferC/OhMyREPL.jl/pull/350/commits/8b6dc6ffbd672039c8cc70724eb3b1c9a2cf0e09#)